src/blockdiagonal.jl

```diff
@@ -143,3 +143,20 @@ function _block_indices(B::BlockDiagonal, i::Integer, j::Integer)
     end
     return p, i, j
 end
+
+
+import Base: copyto!, copy!
```

Comment:
Rather than `import`, please use module qualification to add methods to a function.
```suggestion
```
i.e. below do `function Base.copyto!(...`